### PR TITLE
Document the `framework.router.cache_dir` setting

### PR DESCRIPTION
**Pull request by mpdude**
This documents a new configuration setting suggested in https://github.com/symfony/symfony/pull/47496.

### CODE REVIEW

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -1495,6 +1495,14 @@ If the charset of your application is UTF-8 (as defined in the
...
 errors.
 
+cache_dir
+.........
+
+**type**: ``string`` **default**: ``%kernel.cache_dir%``
+
+The directory where routing information will be cached. Can be set to
+``~`` (``null``) to disable route caching.
+
```
Comment (OskarStark) [inline reference/configuration/framework.rst:1505]: Please add a versionadded directive

Comment (mpdude) [reply reference/configuration/framework.rst:1505]: Added.

**Review by OskarStark — APPROVED** [review]

**Comment by mpdude** [comment]
Code has been merged

**Review by maxhelias — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you Matthias.

**Pull request merged**